docs/getting-started/local-setup/setting-up-a-local-chain.md

```diff
@@ -52,7 +64,214 @@ to delete this directory and start the node up again. If you are using the defau
 
 :::
 
-## Changing the chain ID
+## Starting a local node (manual configuration)
+
+Manually configuring and starting the Gno blockchain node is a bit more involved than simply initializing it "lazily",
+and involves the following steps:
+
+- generating the node secrets, and configuration
+- generating the `genesis.json`, and populating it
+- starting the node with the generated data
+
+### 1. Generate the node directory
+
+You can generate the default node directory using the following command:
+
+```shell
+gnoland init
+```
+
+This will initialize the following directory structure:
+
+```shell
+.
+└── gnoland-data/
+    ├── secrets/
+    │   ├── priv_validator_state.json
+    │   ├── node_key.json
+    │   └── priv_validator_key.json
+    └── config/
+       └── config.toml
+```
+
+A couple of things to note:
+
+- `gnoland init` initializes a default configuration
+- `gnoland init` initializes new node secrets (validator key, node p2p key)
+
+Essentially, `gnoland init` is simply a combination of `gnoland secrets generate` and `gnoland config generate`, with
```

Comment:
Isn't the lazy mode also using the default options?

If so, instead of having a `gnoland init` command, we could focus on providing two options:
1. `gnoland start` (auto mode, full lazy)
2. Detailed documentation explaining how to set up a custom configuration.

What advantage does `gnoland init; gnoland start` offer over simply using `gnoland start`?

Reply:
The main advantage is with the 2 step approach (`gnoland init` + `gnoland start`) users have the freedom to configure things for the client, such as the listen address, peer list, etc, before starting the client

If we went with the single approach of having `gnoland start` with default options (as it is currently), we'd need to add flags for all options in order to keep configurability (not impossible, but highly impractical) - this is why the second suggestion:
> Detailed documentation explaining how to set up a custom configuration.

would require us to always explain how to:
- generate secrets
- generate the config

I honestly believe this 2 step approach is better because it allows us to keep things simple and standard, but also craft more complicated workflows if needed -- this isn't something we have today with `gnoland start`. 

I don't necessarily believe having `gnoland start` be lazy by default is a good idea, primarily because this binary is always going to be part of an orchestration workflow (which requires configurability), and not something users run manually on their machines often. I've never been in a situation with running custom chains where I needed lazy inits, these environments are tightly controlled

The reasoning behind `gnoland init`, which is CLI sugar, is that we wouldn't require a 3 step process for connecting to an existing chain:
- `gnoland secrets generate`
- `gnoland config generate`
- `gnoland start`

This is of course going with the premise that `gnoland start` doesn't lazy init by default

Reply:
@moul 

I've updated the PR description, and refactored this PR to include our discussions:
[Refactor gnoland start to have more specific lazy init](https://github.com/gnolang/gno/pull/1985/commits/e32e72be7deea45b4a85ae534db6900ee4f26c09) and [Update docs](https://github.com/gnolang/gno/pull/1985/commits/d822b648651ec0da7afb9d7757dcaacb8c0405bf)